backend/tracim_backend/lib/core/workspace.py

```diff
@@ -181,22 +166,23 @@ def get_one(self, id):
             ) from exc
 
     def get_one_by_label(self, label: str) -> Workspace:
-        try:
-            return self._base_query().filter(Workspace.label == label).one()
-        except NoResultFound as exc:
+        splitted_label = label.split("~~", maxsplit=1)
+        if len(splitted_label) == 2 and splitted_label[1].isdecimal():
+            return self.get_one(splitted_label[1])
+        else:
+            return self._get_one_by_label(label)
```

Comment:
Please add a note about why code do that

Reply:
Done.